build.xml

```diff
@@ -71,7 +76,7 @@
           resultproperty="test.result"
           failonerror="true">
       <arg value="-c"/>
-      <arg value=". bin/activate;cd ../extension;cfx test ${cfx.binary}"/>
+      <arg value=". bin/activate;cd ../extension;cfx --verbose test ${cfx.binary}"/>
```

Comment:
Hmmm. Not sure what to select as default.

I'm going to adjust the `test` target and the `.travis.yml` too, for now.